client/src/pages/private/ParticipantTable.js

```diff
@@ -166,6 +166,10 @@ export default () => {
 
   const handleSiteSelector = (value) => {
     setSiteSelector(value);
+    setOrder({
+      field: 'distance',
+      direction: 'asc',
```

Comment:
I'm happy we managed to get one-way sorting on this, makes it a lot more usable!